readthedocs/builds/models.py

```diff
@@ -109,6 +129,16 @@ def __str__(self):
                 pk=self.pk,
             ))
 
+    @property
+    def config(self):
+        """Returns the configuration used in the last successful build."""
+        last_build = (
+            self.builds.filter(state='finished', success=True)
+            .order_by('-date')
+            .first()
+        )
+        return last_build.config
```

Comment:
I tried using `values_list` but for some reason, it doesn't return as a dict but as a string, a problem with JSONField I guess

Reply:
https://github.com/dmkoch/django-jsonfield/issues/138

Reply:
Why don't we use the native JSONField from Django?

Reply:
I have already purpose that, but due that we are already using jsonfield in other part of the code we go for using it here too. Also, jsonfield from django is a postgres only feature.